src/main/java/teamproject/CategorySort.java

```diff
@@ -0,0 +1,34 @@
+package teamproject;
+import java.util.ArrayList;
+public class CategorySort implements TestData {
+	//This method is for testing purposes.
+	//Coded with Strings for the time being to focus on the algorithm. Will later switch to structures that include actual recipes.
```

Comment:
Whoops, didn't see this comment until after I reviewed the testdata. Would recommend just fixing the testdata now, or moving this comment down to the testdata file.